graphql/netflix_dgs/src/main/java/com/example/netflixdgs/PostsDataFetcher.java

```diff
@@ -63,19 +30,8 @@ public List<Post> posts() {
             new ParameterizedTypeReference<List<Post>>() {}
```

Comment:
Create one static instance of this variable so you don't intantiate it everytime.